### ISSUE TITLE
slurm - s_p_parse_file: unable to status file /etc/slurm/slurm.conf

### ISSUE THREAD

**Issue by btravouillon** [bluebanquise/bluebanquise#334]
## Description

Jobs fail to start when running as user:
```
[joe@login1 ~]$ srun -N 1 hostname
srun: error: s_p_parse_file: unable to status file /etc/slurm/slurm.conf: Permission denied, retrying in 1sec up to 60sec
srun: error: ClusterName needs to be specified
srun: fatal: Unable to process configuration file
```

The directory /etc/slurm is created with mode 0700:
```
[joe@login1 ~]$ ls -ld /etc/slurm/
drwx------. 2 slurm slurm 4096 Aug 22 22:55 /etc/slurm/
```

The role slurm uses the module `user` to create the slurm user and its home directory /etc/slurm. This module relies on the UMASK defined in /etc/login.defs to set permissions. It does not allow to set the permissions in the role. We might need to add a task to set the permissions accordingly.

## Info
BlueBanquise 1.3-RC2
CentOS Linux release 8.2.2004 (Core)
ansible 2.9.11
slurm 19.05.4

**Comment by jvillemare** [comment]
Closed without resolution? [This solution says that `slurm.conf` needs to be readable by all users](https://groups.io/g/OpenHPC-users/topic/using_slurm_with_other_user/34287187). Copied and pasted here for posterity:

> Your solution is correct.  The proper permissions are discussed in the SLURM documentation[1]
> The slurm.conf file must be readable by all users of Slurm, since it is used by many of the Slurm commands. Other files that are defined in the slurm.conf file, such as log files and job accounting files, may need to be created/owned by the user "SlurmUser" to be successfully accessed. Use the "chown" and "chmod" commands to set the ownership and permissions appropriately. See the section FILE AND DIRECTORY PERMISSIONS for information about the various files and directories used by Slurm.

**Comment by btravouillon** [comment]
That's right, this is the fix I implemented.
GitHub automatically closed this with merge of #337, with a link to the commit as resolution: oxedions closed this in 0ad8955 on 25 Aug

**Comment by jvillemare** [comment]
Ah! My bad, didn't see the `closed this in 0ad8955`. Thank you!
